docs/content/routing/providers/kubernetes-crd.md

```diff
@@ -117,7 +117,7 @@ The Kubernetes Ingress Controller, The Custom Resource Way.
         kind: Rule
         services:
         - name: whoami
-          port: 80
+          port: http
```

Comment:
can we have examples for named port and a number? :)

Reply:
I left the other examples as numbers, I was hoping people would see that you could have both named ports as well as numbered ports. I wasn't sure how to document it without adding a whole other example.